plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/tile.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/tile.hpp"
+
+using LayerTestsDefinitions::TileLayerTest;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+INSTANTIATE_TEST_CASE_P(TileCheck, TileLayerTest,
+                        ::testing::Combine(::testing::Values(std::vector<int64_t>({1})),         //
```

Comment:
If the tiling factor has only ones, this is a no-op; use something larger here, like {3} or {2, 3}